pycbc/io/hdf.py

```diff
@@ -835,3 +835,62 @@ def get_chisq_from_file_choice(hdfile, chisq_choice):
         raise ValueError(err_msg)
     return chisq
 
+def save_dict_to_hdf5(dic, filename):
```

Comment:
As of right now it's not called from anything. But I thought it might be useful in the future for saving the output of the qtransform results.